src/main/java/Duke.java

```diff
@@ -1,10 +1,218 @@
+import exception.EmptyTaskDescException;
+import exception.EmptyTaskTimeException;
+import exception.NoSuchTaskException;
+import tasks.*;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+import java.util.Scanner;
+
 public class Duke {
+
+    public enum TaskTypes {
+        TODO,
+        DEADLINE,
+        EVENT
+    }
+
+    private List<Task> taskList;
+    private SaveData data;
+    private static final File savefile = new File("savedata.txt");
+
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+
+        Duke duke = new Duke();
+        duke.start();
+    }
+
+    public Duke() {
+
+    }
+
+    public void start() {
+        Scanner sc = new Scanner(System.in);
+        System.out.println(
+                "Hello! I'm Duke\n" +
+                "What can I do for you?"
+        );
```

Comment:
Could perhaps break up the line before the operator, in accordance with the coding standards?